clarity_ext/service/file_service.py

```diff
@@ -96,7 +96,7 @@ def queue(self, downloaded_path, artifact, file_prefix=FILE_PREFIX_NONE):
                 file_name = file_name.split("_", 1)[1]
         elif file_prefix == FileService.FILE_PREFIX_RUNNING_NUMBER:
             # Prefix with the index of the shared file
-            artifact_ids = sorted([tuple(map(int, shared_file.id.split("-")) + [shared_file.id])
+            artifact_ids = sorted([tuple(list(map(int, shared_file.id.split("-"))) + [shared_file.id])
```

Comment:
It surprises me if the call to `list` is required if one also calls `tuple` afterwards.

Reply:
I change it.